js/src/Ice/OutgoingConnectionFactory.js

```diff
@@ -200,45 +200,18 @@ export class OutgoingConnectionFactory {
             throw new CommunicatorDestroyedException();
         }
 
-        //
-        // Try to get the connection.
-        //
-        while (true) {
```

Comment:
We don't need a loop here, we are only doing 1 interation.

Reply:
We should fix in Java as well